packages/typo3-guides-cli/src/Repository/Exception/FileException.php

```diff
@@ -26,6 +26,16 @@ public static function notReadable(string $filePath): self
         );
     }
 
+    public static function notWritable(string $filePath): self
```

Comment:
Do we have to introduce our own exceptions for this? It seems like a common issue to me?

Reply:
I like to move the instantiation of an exception into the exception itself (with named constructors). This way it is cleaner and the throwing class does not have to deal with wordings.